src/Microsoft.SqlTools.ServiceLayer/EditData/EditDataService.cs

```diff
@@ -79,6 +79,7 @@ public void InitializeService(ServiceHost serviceHost)
             serviceHost.SetRequestHandler(EditDeleteRowRequest.Type, HandleDeleteRowRequest);
             serviceHost.SetRequestHandler(EditDisposeRequest.Type, HandleDisposeRequest);
             serviceHost.SetRequestHandler(EditInitializeRequest.Type, HandleInitializeRequest);
+            serviceHost.SetRequestHandler(EditRevertCellRequest.Type, HandleRevertCellRequest);
```

Comment:
I'm thinking this is the behavior for if you hit <kbd>esc</kbd> when you're typing a value into a cell. If you revert the entire row, this could be annoying if you need to switch to another tab but are in the middle of changing a row. But this is something we can discuss.